src/components/Accordion/AccordionItem.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Accordion.css';
+
+class AccordionItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.accordionItemToggle = this.accordionItemToggle.bind(this);
+    this.state = {
+      hidden: true,
+    };
+  }
+  accordionItemToggle() {
+    const currentState = this.state.hidden;
+    this.setState({ hidden: !currentState });
+  }
+  render() {
+    return (
+      <li
+        className="p-accordion__group"
+      >
+        <button
+          className="p-accordion__tab"
+          id="status-tab"
+          role="tab"
+          aria-controls="#status"
+          aria-expanded={!this.state.hidden}
+          onClick={() => this.accordionItemToggle(this)}
+          onKeyDown={() => this.accordionItemToggle(this)}
+        >
```

Comment:
I would say the closing bracket should be at the end of the last attr. Is there a reason to wrap it?

Reply:
Linter says closing bracket should match opening bracket.